src/gwt/src/org/rstudio/studio/client/common/filetypes/TextFileType.java

```diff
@@ -288,6 +288,7 @@ public String getScriptInterpreter()
       results.add(commands.goToLine());
       results.add(commands.expandSelection());
       results.add(commands.shrinkSelection());
+      results.add(commands.renameInFile());
```

Comment:
Should we be scoping this to only be available for the R document types? (i.e. stick it in the canExecuteCode() branch along with the other refactoring commands)